services/core-web/common/utils/helpers.js

```diff
@@ -515,5 +515,11 @@ export const formatBooleanToString = (value, defaultValue) => {
 };
 
 export const formatUrlToUpperCaseString = (url) => {
-  return startCase(url.replace("-", " "));
+  const stopWords = ["what", "which", "who", "and", "but"];
```

Comment:
I don't see this function being used in this branch, what is the reason for changing this function?

Reply:
This function is used in InformationRequirementsTableForm.js (line72) to use the URL string to populate the side menu. I had the issue that the word "And" was capitalize. Usually, stop words are not capatalized, so I modified the function to avoid it.
Use of formatUrlToUpperCaseString:
`<Tabs.TabPane
                    tab={formatUrlToUpperCaseString(tab)}
                    key={tab}
                    className="vertical-tabs--tabpane"
                  >
                    {renderTabComponent(this.mergedRequirements, this.state.tabIndex)}
                  </Tabs.TabPane>
                );`